MaterialSkin/HTML/material/html/js/browse-functions.js

```diff
@@ -2979,4 +2986,14 @@ function browseFetchExtra(view, fetchArtists) {
     });
 }
 
+function albumGroupingType(discCount, groupCount, contiguousGroups, parentIsWork) {
```

Comment:
Can you move this function into `uitils-deferred.js` - as its not browse specific (queue also uses this).